RecoParticleFlow/PFClusterProducerAlpaka/src/PFClusterParamsAlpakaESRecord.cc

```diff
@@ -0,0 +1,4 @@
+#include "FWCore/Framework/interface/eventsetuprecord_registration_macro.h"
+#include "RecoParticleFlow/PFClusterProducerAlpaka/interface/PFClusterParamsAlpakaESRecord.h"
+
+EVENTSETUP_RECORD_REG(PFClusterParamsAlpakaESRecord);
```

Comment:
Why do you need a new ES record ?

Reply:
I was primarily following the example of #42807, but taking a closer look at the comments there it seems the suggestion is to reuse the `JobConfigurationGPURecord`?

Reply:
Yes, `JobConfigurationGPURecord` would be fine.

Reply:
The `JobConfigurationGPURecord` should still be moved to some other package from `HeterogeneousCore/CUDACore` (I don't have great suggestions on where though).

Reply:
How about introducing a more general `JobConfigurationRecord` and suggest its usage to all `ESProducers` that do not ave any external sources ?